cmd/actuary/dockerconf_test.go

```diff
@@ -0,0 +1,205 @@
+package actuary
+
+
+import (
+
+	"encoding/json"
+	"testing"
+	"github.com/docker/engine-api/types"
+)
+
+//2. Docker daemon configuration
```

Comment:
just matched it to this:
![screen shot 2017-07-05 at 11 54 04 am](https://user-images.githubusercontent.com/17032867/27880121-b4413756-6178-11e7-9756-ec13d3809599.png)